plex_trakt_sync.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 PATH=/usr/local/bin:/usr/local/sbin:~/bin:/usr/bin:/bin:/usr/sbin:/sbin
-python3 ./main.py
\ No newline at end of file
+(cd ./src && python3 ./main.py)
```

Comment:
you don't need a subshell, there's nothing else after `python` command, so no need to subshell `cd`'ing

in fact. you should use `exec` to release memory from shell process:

```suggestion
cd ./src && exec python3 ./main.py
```

and perhaps just use `set -e` instead of `&&`

```suggestion
see -e
cd ./src
exec python3 ./main.py
```